presto-main/src/test/java/io/prestosql/type/TestVarcharType.java

```diff
@@ -50,4 +55,23 @@ protected Object getGreaterValue(Object value)
     {
         return Slices.utf8Slice(((Slice) value).toStringUtf8() + "_");
     }
+
+    @Test
+    public void testRange()
+    {
+        VarcharType type = createVarcharType(5);
+
+        Type.Range range = type.getRange().get();
+
+        String expectedMax = new StringBuilder()
+                .appendCodePoint(Character.MAX_CODE_POINT)
+                .appendCodePoint(Character.MAX_CODE_POINT)
+                .appendCodePoint(Character.MAX_CODE_POINT)
+                .appendCodePoint(Character.MAX_CODE_POINT)
+                .appendCodePoint(Character.MAX_CODE_POINT)
+                .toString();
+
+        assertTrue(VarcharOperators.equal(Slices.utf8Slice(""), (Slice) range.getMin()));
```

Comment:
I think you can just:
```suggestion
        assertEquals(range.getMin(), utf8Slice(""));
```
(the test can soundly rely on the fact it knows the internal representation; internal representation cannot be changed, it's part of the SPI contract)